### DISCUSSION TITLE
One Chapter at a Time

### DISCUSSION THREAD

**Discussion by rodneyljr** [ezra-bible-app/ezra-bible-app#1158] [Q&A] [answered]
Hi again, I hope it's okay to post these ideas. I don't know how GitHub discussions are supposed to work, but as I use the app, these are just things I think make it better to my needs (and hopefully others). 

When I'm reading a chapter, let's say Matthew 1. I'd love for it to stop at the end of chapter 1, not go automatically to chapter 2. I like this kind of thing for studying one chapter at a time. Just a thought. 

Keep up the great work! Thank you!

**Comment by tobias-klein** [comment, marked as answer]
Hi @rodneyljr
This is already implemented. The option is called "Open all books chapter-wise". Consider the screenshot of the options menu below:
![image](https://github.com/user-attachments/assets/311f75be-f1aa-473e-ae67-8548312005f6)

**Comment by rodneyljr** [comment]
Perfect. Sorry. Must have missed that. Thank you!
